git/cmd.py

```diff
@@ -565,13 +559,11 @@ def is_cygwin(cls) -> bool:
 
     @overload
     @classmethod
-    def polish_url(cls, url: str, is_cygwin: Literal[False] = ...) -> str:
-        ...
+    def polish_url(cls, url: str, is_cygwin: Literal[False] = ...) -> str: ...
```

Comment:
Well, this could be an openioated change...

Reply:
I think this style change is fine and can be accepted. `black` does this now too, starting recently. The reason `@overload`s weren't already formatting this way is that the version of `black` configured for `pre-commit` is older. This is the change related to type stubs, which I mentioned in https://github.com/gitpython-developers/GitPython/issues/1861#issuecomment-1987823048. Before `flake8` was replaced with `ruff`, the disagreement between `black` and `flake8` had held upgrading `black`. Now that `flake8` is replaced, there's nothing stopping this new style from being enforced.